### ISSUE TITLE
Auto dialer with custom dial string.

### ISSUE THREAD

**Issue by catallo** [tschak909/platotermamiga#24]
How about an auto dial function with with configurable dial string? :)

(Assuming that entering the AT command is not part of the original PLATO experience, I don't know because I never had the chance to use a PLATO terminal.)

**Comment by polluks** [comment]
How about an ARexx interface for scripting? Maybe later...

**Comment by catallo** [comment]
Not sure how an ARexx interface would be helpful here ... unless this was a joke?

**Comment by tschak909** [comment]
well i mean, you could use it to send stuff to PLATO. I'm not against
anyone ADDING features if they find a useful thing for it.. I mean, right
now it's possible to cut to the clipboard in both text and graphics (and
pasting is on my list to implement)

-Thom

**Comment by catallo** [comment]
Honestly I'm not sure if anyone would use ARexx with that kind of app ...  but maybe that was polluck's way to communicate that he doesn't consider a dial string an important feature. :)

I was thinking about how to make this more accessible from a user's perspective and the one thing that came to my mind was a virtual keyboard, maybe at the bottom right on the screen, showing those special keys from the PLATO keyboard. It would be great if you could just click on them instead of having to remember those keyboard shortcuts. Just thinking out loud here ...

**Comment by tschak909** [comment]
yes, it can be done, the window is rendered as a gadget, which can be split
up and acted upon.
-Thom

**Comment by polluks** [comment]
@catallo Most terminal apps have a scripting feature.

**Comment by lastebil** [comment]
Put me down for the arexx interface as well.

(With the telser.device, you can just type atdt irata.online:8005 and it would connect - if it wasn't thinking there was no carrier (:  So there essentially _is_ a custom dial string already, though people who are not used to typing AT commands and seeing responses would likely benefit from having it as a menu option.)

**Comment by tschak909** [comment]
is it possible to add AREXX support and still be 1.3 compatible?
-Thom

**Comment by polluks** [comment]
@tschak909 Indeed, ARexx itself does not use 2.0 features.
